### ISSUE TITLE
TypeError: Cannot read property 'setOffset' of undefined

### ISSUE THREAD

**Issue by danielwerg** [susisu/atom-markdown-table-editor#31]
I'm getting this error in console when trying to save markdown file while cursor is outside of a table. Only happens when there is at least one table in a file.

```
Uncaught (in promise) TypeError: Cannot read property 'setOffset' of undefined
    at C:\Users\USERNAME\.atom\packages\markdown-table-editor\node_modules\@susisu\mte-kernel\dist\mte-kernel.js:2835:29
    at C:/Users/USERNAME/.atom/packages/markdown-table-editor/lib/text-editor-interface.js:74:38
    at TextBuffer.transact (C:\Users\USERNAME\AppData\Local\atom\app-1.53.0\resources\app\static\<embedded>:11:493338)
    at TextEditorInterface.transact (C:/Users/USERNAME/.atom/packages/markdown-table-editor/lib/text-editor-interface.js:74:21)
    at TableEditor.formatAll (C:\Users\USERNAME\.atom\packages\markdown-table-editor\node_modules\@susisu\mte-kernel\dist\mte-kernel.js:2763:22)
    at EditorController.formatAll (C:/Users/USERNAME/.atom/packages/markdown-table-editor/lib/editor-controller.js:94:22)
    at C:/Users/USERNAME/.atom/packages/markdown-table-editor/lib/editor-controller.js:46:14
    at Function.simpleDispatch (C:\Users\USERNAME\AppData\Local\atom\app-1.53.0\resources\app\static\<embedded>:11:1190051)
    at C:\Users\USERNAME\AppData\Local\atom\app-1.53.0\resources\app\static\<embedded>:11:1191676
    at Array.map (<anonymous>)
    at Emitter.emitAsync (C:\Users\USERNAME\AppData\Local\atom\app-1.53.0\resources\app\static\<embedded>:11:1191639)
    at TextBuffer.saveTo (C:\Users\USERNAME\AppData\Local\atom\app-1.53.0\resources\app\static\<embedded>:11:498462)
    at async Promise.all (index 0)
```

markdown-table-editor v1.1.2
atom v1.53.0

**Comment by susisu** [comment]
Thank you for reporting!
The error occurs if there is a table at the end of a file.
I will fix it soon.

**Comment by susisu** [comment]
@danielwerg I released v1.1.3 that fixes the error.
Can you try this?

**Comment by danielwerg** [comment]
Just tested it, patch fixed it for me, thanks :)
